packages/ui/src/utils/ThemeUtil.ts

```diff
@@ -628,6 +637,8 @@ function createRootStyles(themeVariables?: ThemeVariables) {
         --wui-avatar-border-base: #252525;
 
         --wui-thumbnail-border-base: #252525;
+
+        --wui-wallet-button-bg-base: var(--wui-color-bg-125);
```

Comment:
I had to add a new css variable for wallet button specifically since `--wui-color-bg-125` wasn't mixing the colors when trying to set custom theme.